includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -1458,9 +1458,7 @@ public function ajax_add_to_cart() {
 			$variation_id = $data_store->find_matching_product_variation( $product, $attributes );
 
 			WC()->cart->add_to_cart( $product->get_id(), $qty, $variation_id, $attributes );
-		}
-
-		if ( in_array( $product_type, [ 'simple', 'variation', 'subscription', 'subscription_variation' ], true ) ) {
+		} elseif ( ! in_array( $product_type, [ 'grouped', 'external' ], true ) ) { // Grouped and external products should not be added to the cart.
```

Comment:
Same here, but maybe makes sense to use`class-wc-stripe-payment-request.php` instead of the newer `class-wc-stripe-express-checkout-helper.php` here (we are deprecating this in the near future anyway):
https://github.com/woocommerce/woocommerce-gateway-stripe/blob/7779798d9031963026f18c09dbbfd428641320ab/includes/payment-methods/class-wc-stripe-payment-request.php#L569-L584

Reply:
Done in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3632/commits/eedd116acc6b98e4bb0e5c10b9e0f934e9df1ab4